api/v1beta1/ssp_webhook.go

```diff
@@ -159,3 +167,16 @@ func validateOperandPlacement(namespace string, placement *api.NodePlacement) er
 
 	return clt.Create(context.TODO(), deployment, &client.CreateOptions{DryRun: []string{metav1.DryRunAll}})
 }
+
+// TODO: also validate DataImportCronTemplates in general once CDI exposes its own validation
+func validateDataImportCronTemplates(r *SSP) error {
+	for _, cron := range r.Spec.DataImportCronTemplates {
+		if len(cron.Name) < 1 {
```

Comment:
Please use `if cron.Name == ""`, because we don't care about the length.

Reply:
Right, old habit. Both are very common ways to check an empty string in Go. Will change.